client/src/components/Donation/components/DonateText.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+
+class DonateText extends Component {
```

Comment:
No need for a component here. A functional component will be better over all as it carries less baggage with it than the `Component` class.

```jsx
function Component(props) {
  return (
    <div>
      <h1>Stateless functional components FTW!</h1>
    </div>
  );
}

// Or with the fancy fat arrow

const Component = props => (
  <div>
    <h1>Stateless functional components FTW!</h1>
  </div>
);
```

This will deliver the exact same as the `Component` you have declared here.

Reply:
Yes. You are correct. This is a leftover of what I was doing while refactoring. Will make it a functional component.